ui/raidboss/data/02-arr/trial/shiva-ex.ts

```diff
@@ -2,13 +2,32 @@ import Conditions from '../../../../../resources/conditions';
 import NetRegexes from '../../../../../resources/netregexes';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export interface Data extends RaidbossData {
+  currentTank: string;
```

Comment:
If you're checking it everywhere, should this be:
```suggestion
  currentTank?: string;
```